ros/src/common/libs/diagnostics_lib/diag_msgs/CHANGELOG.rst

```diff
@@ -0,0 +1,114 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
This change log is not for the diag package.

please create this one for diag packages by using catkin_generate_changelog

Reply:
add CHANGELOG.rst to all packages

Reply:
please add to diag package only.